channeldb/forwarding_package_test.go

```diff
@@ -729,6 +730,49 @@ func TestPackagerSettleFailsThenAdds(t *testing.T) {
 	}
 }
 
+// TestPackagerWipeAll checks that when the method is called, all the related
+// forwarding packages will be removed.
+func TestPackagerWipeAll(t *testing.T) {
+	t.Parallel()
+
+	db := makeFwdPkgDB(t, "")
+
+	shortChanID := lnwire.NewShortChanIDFromInt(1)
+	packager := channeldb.NewChannelPackager(shortChanID)
```

Comment:
That's indeed a good test case, while I don't think it belongs here as a unit test. Maybe there should be some unit tests to ensure we always enforce the scope of a given packager is limited to its channel, with that promise, we could then test all the add/delete/wipe methods of the packager. Otherwise we'd have at least 2x more states to be tested.

Reply:
Yes indeed, I agree it's not a normal unit test scenario, but in this case we're operating on a shared resource, meaning that accidental bug in the way we create the bucket keys may mean unexpected deletes. I'm ok with the test as is and this is more of an bonus we could add.